### PR TITLE
Add parallel algorithms

### PR DESCRIPTION
**Pull request by tupaschoal**
Helps #24

### CODE REVIEW

File: CPP17.md [hunk 1]
```diff
@@ -365,6 +366,17 @@ m.insert(std::move(e));
 // m == { { 1, "one" }, { 3, "three" }, { 4, "two" } }
 ```
 
+### Parallel algorithms
+Many of the STL algorithms, such as the `copy`, `find` and `sort` methods, started to support the so called "parallel execution policies": `seq`, `par` and `par_unseq` which translate to "sequentially", "parallel" and "parallel unsequenced".
```
Comment (AnthonyCalandra) [inline CPP17.md]: Please remove the `so called`. Also make `"parallel execution policies"` italicized and remove the double quotes.

File: CPP17.md [hunk 2]
```diff
@@ -365,6 +366,17 @@ m.insert(std::move(e));
...
+
+```c++
+    std::vector<int> longVector;
+    // Find element using parallel execution policy
+    auto result1 = std::find(std::execution::par, std::begin(longVector), std::end(longVector), 2);
+    // Sort elements using sequential execution policy
+    auto result2 = std::sort(std::execution::seq, std::begin(longVector), std::end(longVector));
```
Comment (AnthonyCalandra) [inline CPP17.md]: Looks like there is some extra indentation here?

**Comment by tupaschoal** [comment]
Done, fixed both issues on all files.

**Comment by tupaschoal** [comment]
No hurries!

**Comment by AnthonyCalandra** [comment]
LGTM, thanks.

**Pull request merged**